src/App.js

```diff
@@ -1,18 +1,19 @@
 import './App.css';
 import React, { useState } from 'react';
-import Todo from './Components/Todo';
+import TodoConnected from './Components/TodoConnected';
```

Comment:
This line is causing some errors, it needs to be:
`./Components/ToDoConnected;`

Reply:
I am on it now, it will be fixed in the next couple of secs:)

Reply:
Ohhh! I pushed this correction to the main branch last night after this PR. SO, it is good on my end:) Currently on App.js there is no `import Todo from './Components/Todo'` on the page. :)